docs/convenience_commands.md

```diff
@@ -93,17 +93,15 @@ phg hvcf2gvcf \
 | `--batch-size`       | Number of sample vcf files to export in a single batch from tiledb                                                  | `5`                                |                  |
 
 
-### Create a GFF file from an imputed H.vcf file
+### Create a GFF file from an imputed hVCF file
+
+> Create a _path-specific_ [GFF file](https://en.wikipedia.org/wiki/General_feature_format) 
+> from an imputed hVCF file and existing sample GFF files.
```

Comment:
Do you think saying "path-specific GFF" is enough? Or should we include the additional documentation on why a user might want this, ie the sentence (or a version of this sentence) "This command is useful for creating a GFF file that contains only the annotations for the paths that are present in the imputed H.vcf file".

Reply:
I think the additional comment is helpful. In my opinion, it's not immediately intuitive what the cross-section between a gff and an hvcf is, so more context is warranted.

Reply:
Updated